CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ option(SSG_PRODUCT_FIREFOX "If enabled, the Firefox SCAP content will be built"
 option(SSG_PRODUCT_JBOSS_EAP6 "If enabled, the JBoss EAP6 SCAP content will be built" TRUE)
 option(SSG_PRODUCT_JBOSS_FUSE6 "If enabled, the JBoss Fuse6 SCAP content will be built" TRUE)
 option(SSG_PRODUCT_JRE "If enabled, the JRE SCAP content will be built" TRUE)
+option(SSG_PRODUCT_OCP3 "If enabled, the RHOCP3 SCAP content will be built" TRUE)
```

Comment:
Why not set this to FALSE?

Reply:
@yuumasato this was to stay consistent with how `RHEVM3` was disabled.

Reply:
Oh, ok, it is not a big deal.